sympy/utilities/lambdify.py

```diff
@@ -450,6 +455,44 @@ def wrapper(*argsx, **kwargsx):
                     "Expression:\n\n{expr}").format(sig=sig, expr=expr_str)
     return func
 
+def should_dummify(expr, printer=None):
+    """
+    Returns the value of dummify to be used. This function either returns True or False.
+
+    Parameter:
```

Comment:
Can you please elaborate on your suggestion? . I looked at the link you have provided but couldn't get what change. Thanks